### DISCUSSION TITLE
Monster Browser 0.9f

### DISCUSSION THREAD

**Discussion by torhus** [torhus/monster-browser#35] [Announcements]
- New: Filter servers on name and game/gamename cvars, or on player names.
- Change: Game selection dropdown is no longer editable, use the new filter feature instead.

<hr /><em>This discussion was created from the release <a href='https://github.com/torhus/monster-browser/releases/tag/v0.9f'>Monster Browser 0.9f</a>.</em>

**Comment by torhus** [comment]
![MB09f](https://user-images.githubusercontent.com/19881605/195931317-bad613c7-e9ad-4dc8-b0d5-65dfe0178372.png)

**Comment by The-Gig** [comment]
Tried. The new search seems to work well!
